packages/element-app/package.json

```diff
@@ -5,25 +5,28 @@
   "homepage": "https://element-did.com",
```

Comment:
Adding the BIP39 key or the keyagreement key does not work.

 Let's filter out keys that cannot be added in this dialog. (Btw why do we have this type of key?)

<img width="545" alt="Screenshot 2020-01-24 at 14 26 28" src="https://user-images.githubusercontent.com/7913565/73051591-94857b80-3eb5-11ea-8a7d-1de3a86da83f.png">